great_expectations/datasource/fluent/sql_datasource.py

```diff
@@ -930,7 +987,7 @@ def as_selectable(self) -> sqlalchemy.Selectable:
 
         This can be used in a from clause for a query against this data.
         """
-        return sa.text(self.qualified_name)  # type: ignore[return-value]
+        return sa.table(self.table_name, schema=self.schema_name)
```

Comment:
The return type didn't match the annotated return type. I actually need a selectable for my new check to work. I haven't followed up with how/if we were using this mis-annotated return value before and will update code/tests if necessary.